src/EditorFeatures/Core/InlineDiagnostics/InlineDiagnosticsOptions.cs

```diff
@@ -9,11 +9,11 @@ namespace Microsoft.CodeAnalysis.Editor.InlineDiagnostics
     internal sealed class InlineDiagnosticsOptions
     {
         public static readonly PerLanguageOption2<bool> EnableInlineDiagnostics =
-            new("InlineDiagnosticsOptions_EnableInlineDiagnostics",
+            new("dotnet_enable_inline_diagnostics",
                 defaultValue: false);
 
         public static readonly PerLanguageOption2<InlineDiagnosticsLocations> Location =
-            new("InlineDiagnosticsOptions_Location",
+            new("dotnet_inline_diagnostics_location",
                 defaultValue: InlineDiagnosticsLocations.PlacedAtEndOfCode, serializer: EditorConfigValueSerializer.CreateSerializerForEnum<InlineDiagnosticsLocations>());
```

Comment:
Would be nice to merge this into a single option (location == none would mean disabled). But I guess not worth changing now.